lib/core-integration/src/Test/Integration/Scenario/API/Shelley/StakePools.hs

```diff
@@ -253,6 +254,7 @@ spec = do
             verify rWithdrawal
                 [ expectResponseCode HTTP.status200
                 , expectField #withdrawals (`shouldSatisfy` (not . null))
+                , expectField #amount (`shouldBe` totalAmtAfterWithdrawals)
```

Comment:
so here basically we check that amount in tx with withdrawal is the same as tx goes from Pending -> InLedger